### DISCUSSION TITLE
How to connect the results to whisper ?

### DISCUSSION THREAD

**Discussion by amitli1** [juanmc2005/diart#141] [General]
Is it possible to connect the results of `diart` to `whisper` ?

For example, instead of outputting the results to `rttm` file, can we output the results to  `whisper` ASR model ?

How can I do it ?

**Comment by juanmc2005** [comment]
Hi @amitli1, absolutely! I'm currently preparing a tutorial to do so, stay tuned!

**Reply by amitli1** [reply]
thanks it will be very helpful

**Reply by benjamin-kirkbride** [reply]
@juanmc2005 please update here when you do so we can see it!

**Reply by juanmc2005** [reply]
Will do! You can also follow me on twitter for diart updates and tutorials @juanmc2005

**Comment by juanmc2005** [comment]
The tutorial is now live [here](https://medium.com/@juanmc2005/color-your-captions-streamlining-live-transcriptions-with-diart-and-openais-whisper-6203350234ef)! Let me know what you think!

**Reply by amitli1** [reply]
thanks a lot
